pom.xml

```diff
@@ -199,7 +199,7 @@
 
         <aerogear.bom.version>0.2.20.2</aerogear.bom.version>
 
-        <kafka.clients.version>0.10.2.1</kafka.clients.version>
+        <kafka.clients.version>0.11.0.0</kafka.clients.version>
```

Comment:
what do we gain here ? 
Is that really compatible?  

I guess we need to update cdi library, and the DZB related changes too - if they are already on .11